Can't build due to "Argument list too long" error

When trying to run my app using `react-native run-ios`, I receive the following error:

```
=== BUILD TARGET RNAdMobManager OF PROJECT RNAdMobManager WITH CONFIGURATION Debug ===


Check dependencies

Argument list too long: recursive header expansion failed at /Users/gilad/MyProject/node_modules/react-native-admob/ios/../../../ios/Pods/React/node_modules/fsevents/node_modules/ajv/dist.



** BUILD FAILED **
```

I have the following in my `package.json`:
```
    "react-native-admob": "^2.0.0-beta.2",
```

And inside my `Podfile`:
```
  pod 'Google-Mobile-Ads-SDK'
```

I've tried to clean my `build` folder, as well as XCode's `DerivedData` - but it doesn't help...

--- a/ios/RNAdMobManager.xcodeproj/project.pbxproj
+++ b/ios/RNAdMobManager.xcodeproj/project.pbxproj
@@ -247,7 +247,7 @@
 		A96DA77E1C146D7200FC639B /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				FRAMEWORK_SEARCH_PATHS = "$(PROJECT_DIR)/../../../ios/**";
+				FRAMEWORK_SEARCH_PATHS = "$(PROJECT_DIR)/../../../ios/Pods/**";
 				HEADER_SEARCH_PATHS = (
 					"$(inherited)",
 					"$(SRCROOT)/../../../node_modules/react-native/React/**",
@@ -261,7 +261,7 @@
 		A96DA77F1C146D7200FC639B /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				FRAMEWORK_SEARCH_PATHS = "$(PROJECT_DIR)/../../../ios/**";
+				FRAMEWORK_SEARCH_PATHS = "$(PROJECT_DIR)/../../../ios/Pods/**";
 				HEADER_SEARCH_PATHS = (
 					"$(inherited)",
 					"$(SRCROOT)/../../../node_modules/react-native/React/**",

